pinot-integration-tests/src/test/java/org/apache/pinot/compat/tests/SegmentOp.java

```diff
@@ -36,15 +57,23 @@
  */
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class SegmentOp extends BaseOp {
+  private static final Logger LOGGER = LoggerFactory.getLogger(SegmentOp.class);
+  private static final FileFormat DEFAULT_FILE_FORMAT = FileFormat.CSV;
+  private static final String STATE_ONLINE = "ONLINE";
```

Comment:
Please pick this up from CommonConstants

Reply:
Sure.

Reply:
Fixed